### PR TITLE
WabiSabi subpage

### PR DESCRIPTION
**Pull request by CAnorbo**

### CODE REVIEW

File: wwwroot/integrations.html [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+            <section class="p-4">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-12 col-sm-8 offset-sm-2 text-center">
+                            <div class="my-5">
+                                <h2 class="pt-header title-lined">Eligibility Criteria</h2>
+
+                                <p>Businesses and developers who want their Bitcoin products eligible to integrate the coinjoin API may be asked to meet certain technical requirements around network-level privacy, such as using block filters and Tor.</p>
+
+                                <a href="#" class="btn btn-link">Read more →</a>
```
Comment (MarnixCroes) [inline wwwroot/integrations.html:224]: I think this should link somewhere else.
to the blog, like on zksnacks website?

**Review by yahiheb — APPROVED** [review]
cACK

File: wwwroot/integrations.html (hunk 1 above)
Comment (CAnorbo) [reply wwwroot/integrations.html]: Good catch! Fixed.

File: wwwroot/index.html [hunk 2]
```diff
@@ -252,6 +255,20 @@ <h2 class="title title-lined">DOWNLOAD<br>WASABI WALLET</h2>
...
+                        <h2 class="title-small">WABISABI<br>SIMPLE & POWERFUL COINJOIN API</h2>
+                        <p>Enterprises can protect their users' privacy with collaborative Bitcoin transactions.</p>
+                        <a href="integrations.html" class="btn btn-primary" target="_blank">READ MORE</a>
+                    </div>
+                    <div class="col-xl-4 offset-xl-1 oreder-1 order-xl-2 text-center">
+                        <img src="img/CJ_v0003.png" class="integration-img">
+                    </div>
+                </div>
+            </div>
+        </section>
```
Comment (MarnixCroes) [inline wwwroot/index.html:271]: not sure if this should be at `THE WALLET` subpage

**Review by MaxHillebrand — APPROVED** [review]

**Comment by MarnixCroes** [comment]
@CAnorbo should the `Integrations` subpage have a link to zksnacks website?

**Comment by CAnorbo** [comment]
> @CAnorbo shouldn't the `Integrations` subpage have a link to zksnacks website?

We have exactly the same content here, so I don't see the point. What is your reason?

**Comment by MarnixCroes** [comment]
> > @CAnorbo shouldn't the `Integrations` subpage have a link to zksnacks website?
> 
> We have exactly the same content here, so I don't see the point. What is your reason?

just a question. If you think it's ok it's ok

**Review by MarnixCroes — APPROVED** [review]
tack

**Comment by MarnixCroes** [comment]
Benefits of the coinjoin api icons are not ok.
If you run it locally it's fine, but just checked on the website and it's different.

[Screencast from 24-11-2023 10:37:39.webm](https://github.com/zkSNACKs/WasabiWalletWebSite/assets/93143998/74225944-056f-4b40-bf00-f777d8edcb2f)

locally
image

**Comment by kristapsk** [comment]
Purged CF cache, that solved problem for me.

**Comment by CAnorbo** [comment]
Yes, we had to purge the cache on CF, but know everything seems good.

**Comment by MarnixCroes** [comment]
> Purged CF cache, that solved problem for me.

ok yes seems to be a local problem

**Comment by yahiheb** [comment]
Is calling this `Integrations` better than `Coinjoin API`? I think the latter is more straight forward.

**Pull request merged**